src/ports/items/utils.ts

```diff
@@ -192,7 +192,7 @@ export function getItemsQuery(filters: ItemFilters, isCount = false) {
     where.push(`searchText_contains: "${search.trim().toLowerCase()}"`)
   }
 
-  if (contractAddresses && contractAddresses.length > 0) {
```

Comment:
Is this intentional? Careful here, `contractAddresses` will always be `[]` even tho TS says it could be `undefined`. 
This is because how it's declared: `const contractAddresses = params.getList('contractAddress')`, the `getList` will always return an array (empty if the param is not passed). That said, I think we need the `&& contractAddresses.length > 0` here.

Reply:
Yes its necessary, I removed it because I was trying things and forgot to add it back. Thanks!

Reply:
👍